modules/telco-ran-crs-day-2-operators.adoc

```diff
@@ -13,9 +13,9 @@ Component,Reference CR,Description,Optional
 Cluster Logging Operator,`ClusterLogForwarder.yaml`,Configures log forwarding for the cluster.,No
 Cluster Logging Operator,`ClusterLogNS.yaml`,Configures the namespace for cluster logging.,No
 Cluster Logging Operator,`ClusterLogOperGroup.yaml`,Configures Operator group for cluster logging.,No
-Cluster Logging Operator,`ClusterLogServiceAccount.yaml`,New in 4.18. Configures the cluster logging service account.,No
-Cluster Logging Operator,`ClusterLogServiceAccountAuditBinding.yaml`,New in 4.18. Configures the cluster logging service account.,No
-Cluster Logging Operator,`ClusterLogServiceAccountInfrastructureBinding.yaml`,New in 4.18. Configures the cluster logging service account.,No
+Cluster Logging Operator,`ClusterLogServiceAccount.yaml`,Configures the cluster logging service account.,No

```

Comment:
moving into 4.19, we probably don't need to highlight this?
@imiller0

Reply:
Correct. It is still part of the reference in 4.19, but not "new".

Reply:
@slovern so does this line need removal?